src/mpi/coll/include/coll_impl.h

```diff
@@ -178,41 +178,41 @@ typedef enum MPIR_Iallgather_intra_algo_t {
     MPIR_IALLGATHER_INTRA_ALGO_AUTO,
     MPIR_IALLGATHER_INTRA_ALGO_BRUCKS,
     MPIR_IALLGATHER_INTRA_ALGO_RECURSIVE_DOUBLING,
-    MPIR_IALLGATHER_INTRA_ALGO_RING
+    MPIR_IALLGATHER_INTRA_ALGO_RING,
```

Comment:
I honestly don't care enough to move this kind of thing to separate files, but it probably should be mentioned as a second cleanup in the commit message.